pkg/kanctl/kanctl.go

```diff
@@ -39,6 +39,12 @@ var (
 	Verbose bool
 )
 
+type Clients struct {
+	KanisterClient kubernetes.Interface
+	CrdClient      versioned.Interface
+	OSClient       osversioned.Interface
+}
```

Comment:
```suggestion
type Clients struct {
	KubeClient kubernetes.Interface
	KanisterClient      versioned.Interface
	OSClient       osversioned.Interface
}
```
Let's rename CrdClient to KanisterClient

Reply:
https://github.com/kanisterio/kanister/pull/1666#discussion_r992614443
I think there is confusion. In this comment, Ivan suggested renaming CrdClient not KubeClient :)

Reply:
Understood, thanks!